paths.scad

```diff
@@ -872,12 +872,12 @@ function assemble_a_path_from_fragments(fragments, rightmost=true, startfrag=0,
         // Found fragment is already closed
         [foundfrag, concat([path], remainder)]
     ) : let(
-        fragend = select(foundfrag,-1),
+        fragend = last(foundfrag),
         hits = [for (i = idx(path,e=-2)) if(approx(path[i],fragend,eps=eps)) i]
     ) hits? (
         let(
             // Found fragment intersects with initial path
-            hitidx = select(hits,-1),
+            hitidx = last(hits),
             newpath = list_head(path,hitidx),
             newfrags = concat(len(newpath)>1? [newpath] : [], remainder),
             outpath = concat(slice(path,hitidx,-2), foundfrag)
```

Comment:
slice call needs update?

Reply:
Actually, I changed this one back to `slice()`, just in case `hitidx` is ever that near the end of the list.